tensorboard/data/provider.py

```diff
@@ -52,6 +52,25 @@ def data_location(self, experiment_id):
         """
         return ""
 
+    def list_active_plugins(self, experiment_id):
```

Comment:
Maybe just `list_plugins()`?  Given that this is scoped to the experiment already, I'm not sure that "active" as a qualifier really means much in the data provider context?